src/handlebars/support.handlebars

```diff
@@ -55,13 +55,16 @@
         The frequency of AdoptOpenJDK releases is guided by the schedule of our dependencies.
         We produce builds based upon source code at <a href="https://openjdk.java.net" target="_blank">OpenJDK</a>,
         <a href="https://www.eclipse.org/openj9/" target="_blank">Eclipse OpenJ9</a>,
+        <a href="https://github.com/corretto" target="_blank">Amazon Corretto</a>,
```

Comment:
The page for the general public about Corretto is at https://aws.amazon.com/corretto/

Reply:
Fixed